tests/beaker/tests/create_vg_w_param_name.rb

```diff
@@ -0,0 +1,47 @@
+require 'master_manipulator'
+require 'lvm_helper'
+require 'securerandom'
+
+test_name "FM-4579 - C96594 - create volume group without parameter 'name'"
```

Comment:
The test actually creates a volume group with a name. Typo in the test name title?